sklearnex/ensemble/tests/test_forest.py

```diff
@@ -44,3 +44,30 @@ def test_sklearnex_import_rf_regression():
             or np.allclose([-8.36], pred, atol=1e-2)
     else:
         assert_allclose([-6.66], pred, atol=1e-2)
+
+
+def test_sklearnex_import_et_classifier():
+    from sklearnex.ensemble import ExtraTreesClassifier
+    X, y = make_classification(n_samples=1000, n_features=4,
+                               n_informative=2, n_redundant=0,
+                               random_state=0, shuffle=False)
+    #For release 2023.2, random_state isn't implemented
+    #defaults to seed=777 although it is set to 0
+    rf = ExtraTreesClassifier(max_depth=2, random_state=0).fit(X, y)
```

Comment:
Add warning for unsupported `random_state` parameter

Reply:
Sounds good. Let me know if we should also make a ticket for similar warnings to the sklearnex onedal SVM.